eng/Versions.props

```diff
@@ -33,7 +33,7 @@
     <!-- TargetingPackVersionPrefix is used by projects, like .deb and .rpm, which use slightly different version formats. -->
     <TargetingPackVersionPrefix>$(VersionPrefix)</TargetingPackVersionPrefix>
     <!-- Targeting packs do not produce patch versions in servicing builds. No API changes are allowed in patches. -->
-    <TargetingPackVersionPrefix Condition="'$(IsTargetingPackBuilding)' != 'true'">$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).10</TargetingPackVersionPrefix>
+    <TargetingPackVersionPrefix Condition="'$(IsTargetingPackBuilding)' != 'true'">$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).13</TargetingPackVersionPrefix>
```

Comment:
Would arm64 TP package have *.*.0 version?

Reply:
We want to build just the arm-64 .msi, with a 3.1.0 version - we don't want to build the .nupkg

Reply:
@NikolaMilosavljevic Historically, the version of the installer and targeting pack follows whichever servicing patch built it. This is why in the official 3.1.406 SDK, you'll see that the targeting pack is installed under packs\Microsoft.AspNetCore.App.Ref\3.1.10. 

@wtgodbe I tried but couldn't easily split the logic controlling building the targeting pack nupkg and the installers. They are all based on `IsTargetingPackBuilding`. In fact, the installer build extracts the zip of the targeting pack that was built when `IsPackable` is true in the Microsoft.AspNetCore..App.Ref project, which means it also builds the nupkg. I don't want to add additional logic to handle this scenario and I am under the impression that the SDK can choose to ingest just the msi installers and ignore the nupkg. Is that possible @NikolaMilosavljevic?

Reply:
We could choose not to ingest the new .nupkg in dotnet/installer, but that's manual and risks us messing it up - we'd also have to manually remove it from the drop so it doesn't get published to nuget.org. Maybe the best thing would be to just not publish it - you could add a post-build target that deletes the .nupkg from `ArtifactsShippingPackagesDir` before the manifest gets generated, for example.